src/reducerFn.js

```diff
@@ -14,21 +14,20 @@ export default function reducerFn(initialState, actions={}, reducer) {
   const { actionFetch, actionSuccess, actionFail,
     actionReset, actionCache, actionAbort } = actions;
   return (state=initialState, action)=> {
+    const params = action.params || {};
     switch (action.type) {
       case actionFetch:
         return {
           ...state,
-          pathvars:{},
-          body:{},
+          pathvars: action.pathvars || {},
```

Comment:
Same here. Pathvars are stored in action.request.pathvars. This is currently defaulting.